com.unity.render-pipelines.universal/Runtime/2D/Renderer2DData.cs

```diff
@@ -76,8 +63,8 @@ public enum Renderer2DDefaultMaterialType
         [SerializeField, Reload("Runtime/Data/PostProcessData.asset")]
         PostProcessData m_PostProcessData = null;
 
-        public float hdrEmulationScale => m_HDREmulationScale;
-        public Light2DBlendStyle[] lightBlendStyles => m_LightBlendStyles;
+        internal float hdrEmulationScale => m_HDREmulationScale;
+        internal Light2DBlendStyle[] lightBlendStyles => m_LightBlendStyles;
```

Comment:
These changes may fail the package validation tests.

Reply:
how do we remove API?

Reply:
I think we need to first mark them as obsolete, then remove them in a later major version.

Reply:
I've reverted this change for now. We will do a holistic review of public APIs before we get out of preview.